server/app/rss.py

```diff
@@ -0,0 +1,17 @@
+import feedparser
+from .watchlist import Watchlist
+
+def get_rss_url(channel_id):
+	return f'https://www.youtube.com/feeds/videos.xml?channel_id={channel_id}'
+
+def from_file(filename):
+	return feedparser.parse(filename)
+
+def from_channel_id(channel_id):
+	print(channel_id)
+	return feedparser.parse(get_rss_url(channel_id))
```

Comment:
We need to get this data into a cleaned up json object. Currently, this gives a very noisy object with a lot of duplicate information and inconsistent naming patterns (title vs yt_videoid).  We should return a list of json objects with a handful of clear and relevant properties.

![image](https://user-images.githubusercontent.com/14893203/162787225-51ef4678-5a8b-4057-9c16-28a52de39877.png)